playbook/app/pb_kits/playbook/pb_card/docs/_card_swift.md

```diff
@@ -0,0 +1,30 @@
+### Description
+Card component can easily wrap custom content using the same Playbook pattern.
+
+### When to Use
+- Render content with same visual style and pattern
+- Organize content with Playbook separator style 
+- Render content with Playbook background colors
+
+### When not to Use
+- To execute actions and for tappble areas, try to use buttons instead.
+
+### How to use
+| Code | Component |
+| --------- | --------- |
+| <img width="401" alt="Screenshot 2023-06-15 at 21 16 28" src="https://github.com/powerhome/playbook/assets/60269827/def5e7ac-4482-4847-89a9-98c5cb182620"> | <img width="401" alt="Screenshot 2023-06-15 at 21 15 04" src="https://github.com/powerhome/playbook/assets/60269827/36977781-0dfa-4928-9d2d-63851e8e09bd"> |
```

Comment:
We should agree on what the screenshots should look like, currently mine are taken from the markdown file while yours are taken from xcode it seems. Take a look at my documentation [here](https://pr2603.playbook.beta.px.powerapp.cloud/kits/textarea/swift).